changelog/unreleased/kong/kong_request_id.yml

```diff
@@ -0,0 +1,3 @@
+message: Bumped lua-kong-nginx-module from 0.7.1 to 0.8.0
```

Comment:
The filename of this file should be changed. Maybe `bump_lua_kong_nginx_module.yml`

Reply:
done.

Reply:
We already have a `lua_kong_nginx_module_bump.yml` in PR #11663, should we rename it to another name?

Reply:
How about `bump_lua_kong_nginx_module_from_0_7_1_to_0_8_0.yml`?

Reply:
reuse that bump yml file.